.gitignore

```diff
@@ -63,4 +63,5 @@ build/
 
 mpi-proxy-split/mpi-wrappers/p2p-deterministic.h
 mpi-proxy-split/mpi-wrappers/mana_p2p_update_logs
-manpages/mana.1.gz 
\ No newline at end of file
+restart_plugin/lower_half_api.h
```

Comment:
Please remove this `restart_plugin/lower_half_api.h`.  Probably it's a stray file in your own build.  MANA does not create this file.

Reply:
@gc00 
MANA does create this file. See this line in MANA's root `Makefile.in`: https://github.com/mpickpt/mana/blob/56d5b494730eb7ed757e32528bbbba16104f79dd/Makefile.in#L56C1-L56C1

Either the copying is not required or we need to ignore the file.

Reply:
Then probably it's fine to add it to .gitignore.  I stand corrected.  Please do include it the way you have it.